daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -835,6 +843,18 @@ func getDeviceUUID(device string) (string, error) {
 	return uuid, nil
 }
 
+func getDeviceFS(device string) (string, error) {
```

Comment:
Right. Only difference seems to be that ProbeFsType() will return error if fs is anything other then ext4, xfs and btrfs. As we already use ProbeFsType(). So I can use ProbeFsType() too instead of using blkid.